php/example_code/sns/README.md

```diff
@@ -0,0 +1,35 @@
+# Amazon Simple Notification Service (SNS) AWS SDK for PHP - Version 3 code examples
+
+## Purpose
+The code examples in this directory demonstrate how to work with the  Amazon Simple Notification Service 
+(Amazon SNS) using the AWS SDK for PHP version 3 (v3).
+
+Amazon SNS is a fully managed messaging service for both system-to-system and app-to-person (A2P) communication. 
+
+## Code examples
+This is a workspace where you can find AWS SDK for PHP v3 Amazon SNS examples.
+
+- [Check phone opt out](./CheckOptOut.php)
+- [Confirm a subscription](./ConfirmSubscription.php)
+- [Create a topic](./CreateTopic.php)
+- [Delete a topic](./DeleteTopic.php)
+- [Get SMS attributes](./GetSMSAttributes.php)
+- [Get topic attributes](./GetTopicAttributes.php)
+- [List opted out numbers](./ListOptOut.php)
+- [List subscriptions](./ListSubscriptions.php)
+- [List topics](./ListTopics.php)
+- [Publish SMS](./PublishTextSMS.php)
+- [Publish to topics](./PublishTopic.php)
+- [Set SMS type](./SetSMSAttributes.php)
+- [Set topic attributes](./SetTopicAttributes.php)
+- [Subscribe to an email](./SubscribeEmail.php)
+- [Subscribe to an HTTPS endpoint](./SubscribeHTTPS.php)
+- [Subscribe to Lambda](./SubscribeLambda.php)
+- [Subscribe to SMS](./SubscribeTextSMS.php)
+- [Unscribe](./unsubscribe.php)
+
+## Resources
+- [AWS SDK for PHP - Version 3](https://github.com/aws/aws-sdk-php) 
```

Comment:
Seems a little self-referential...do we want to link to GitHub from GitHub like this?
Also, remove the dash from the title.